integration/scripts/integration.sh

```diff
@@ -25,17 +25,17 @@ usage() {
 }
 
 build_containers() {
-    bash -c "${DC} -f docker-compose.yml -f docker-compose.xpu.yml -f docker-compose.otel.yml -f docker-compose.spdk.yml -f docker-compose.pxe.yml build --parallel"
+    bash -c "${DC}  build --parallel"
```

Comment:
Giant nit (but worth fixing) is the extra space you left in these lines.

Reply:
fixed